### PR TITLE
Feat sign up login page

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the Signup and Login Page
Screenshot (558)
Screenshot (559)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -203,6 +208,18 @@
 .flex-end{
     justify-content: flex-end;
 }
+.maple-flex-col-xy-center{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items:center;
```
Comment (Sameony) [inline src/App.css:215]: i think you had a utility class for the properties in line 212,214,215, so maybe you could reuse that and refactor the code later on, as i can see lots of redundant code in the css.

**Review by Sameony — APPROVED** [review]
good commit messages and clean code. do follow up with refactoring afterwards

File: src/App.css (hunk 1 above)
Comment (krishanu2911) [reply src/App.css:215]: Thank you, Sammer.
I will Refactor the CSS

**Review by Priyanshi-Tiwari — APPROVED** [review]
Everything looks good to me. I found the code clean and easy to understand. Good work.

**Review by shubhamkumar648 — APPROVED** [review]
Good Folder Structure.
Descriptive Class Names.
Clean and understandable code.
Good To merge

**Pull request merged**